### PR TITLE
iTerm SSH Dropdown Menu

### PR DESCRIPTION
**Pull request by magicdude4eva**
Provides a simple menu of ssh hosts when clicked on opens a new ssh session in terminal

### CODE REVIEW

File: Network/iTermSshMenu.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# <bitbar.title>iTerm SSH Dropdown Menu</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Gerd Naschenweng</bitbar.author>
+# <bitbar.author.github>magicdude4eva</bitbar.author.github>
+# <bitbar.desc>Provides a simple menu of ssh hosts when clicked on opens a new ssh session in terminal</bitbar.desc>
+# <bitbar.image></bitbar.image>
```
Comment (matryer) [inline Network/iTermSshMenu.sh:8]: Please take a screenshot of it running and include an image. It makes the plugin more likely to be seen and used.

**Review by matryer — CHANGES_REQUESTED** [review]

File: Network/iTermSshMenu.sh (hunk 1 above)
Comment (magicdude4eva) [reply Network/iTermSshMenu.sh:8]: I hope this is fine:
image


And to see it running:
iTermDropdown

**Comment by willhumble** [comment]
Brilliant!

**Review by magicdude4eva — COMMENTED** [review]
done - got somewhat stuck

**Comment by sprak3000** [comment]
@magicdude4eva 

Could you 

1. Provide a screenshot in the `<xbar.image>` metadata tag
2. Update the other metadata tags from `<bitbar...>` to `<xbar...`
3. Run `chmod +x` on the file

**Review by sprak3000 — APPROVED** [review]
Will `chmod` in separate PR...

**Pull request merged**